modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationQueueEntryResourceImpl.java

```diff
@@ -29,4 +45,124 @@
 )
 public class NotificationQueueEntryResourceImpl
 	extends BaseNotificationQueueEntryResourceImpl {
+
+	@Override
+	public void deleteNotificationQueueEntry(Long notificationQueueEntryId)
+		throws Exception {
+
+		_notificationQueueEntryService.deleteNotificationQueueEntry(
+			notificationQueueEntryId);
+	}
+
+	@Override
+	public NotificationQueueEntry getNotificationQueueEntry(
+			Long notificationQueueEntryId)
+		throws Exception {
+
+		return _toNotificationQueueEntry(
+			_notificationQueueEntryService.getNotificationQueueEntry(
+				notificationQueueEntryId));
+	}
+
+	@Override
+	public Page<NotificationQueueEntry> getNotificationQueueEntryPage(
+			String search, Aggregation aggregation, Filter filter,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		return SearchUtil.search(
+			HashMapBuilder.put(
+				"get",
+				addAction(
+					ActionKeys.VIEW, "getNotificationQueueEntryPage",
+					NotificationConstants.RESOURCE_NAME,
+					contextCompany.getCompanyId())
+			).build(),
+			booleanQuery -> {
+			},
+			filter,
+			com.liferay.notification.model.NotificationQueueEntry.class.
+				getName(),
+			search, pagination,
+			queryConfig -> queryConfig.setSelectedFieldNames(
+				Field.ENTRY_CLASS_PK),
+			searchContext -> {
+				searchContext.setAttribute(Field.NAME, search);
```

Comment:
@p-albuquerque There's no name for NotificationQueueEntry. We can remove this row.